<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_ImagenCliente1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_Cliente1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenCliente2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_Cliente2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenCliente3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_Cliente3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenBalanceador">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_Balanceador</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenFabricaA">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_FabricaA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenFabricaB">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_FabricaB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenFabricaC">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_FabricaC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenFabricaExtra">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_FabricaExtra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenColaDeAlistadores">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_ColaDeAlistadores</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenFactura">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_Factura</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImagenEmpacador">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_Empacador</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
